index.html

```diff
@@ -32,82 +32,57 @@
         </nav>
     </header>
 
-    <main class="main">
-        <section class="section">
-            <div class="icons">
-                <ul>
-                    <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-m"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
-                </ul>
-            </div>
-            <div class="details">
-                <div class="info">
-                    <div class="title">
-                        <h1>I'm Precious <br> Glad to see you!</h1>
-                    </div>
-                    <div class="text">
-                        <p>I’m a software developer! I can help you build a product , feature or website Look through
-                            some of my work and experience! If you like what you see and have a project you need coded,
-                            don’t hestiate to contact me.
-                        </p>
-                    </div>
-                    <p class="con">LET'S CONNECT</p><br>
+    <section class="section">
+        <div class="icons">
+            <ul>
```

Comment:
- [ ] As the previous reviewer suggested, Kindly work on the border radius of this particular section to match the original design

![image](https://user-images.githubusercontent.com/98453979/194707888-0c99de00-9539-4ea0-beef-3ea9c8331080.png)

### Checkout figma

![image](https://user-images.githubusercontent.com/98453979/194707907-9bb66049-c840-4728-bb95-e5a298d6f70e.png)

- [ ] Kindly work on the padding of this section to match the Figma design. There should be some space around the container

![image](https://user-images.githubusercontent.com/98453979/194708099-3b85167d-bef5-4e92-8b47-99a0c78aea3c.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/194708130-425d7fbf-1277-44af-b5ea-15ab1ca0c0c5.png)